test/spectests/spectests.cpp

```diff
@@ -576,34 +576,25 @@ bool run_tests_from_dir(const fs::path& path, const test_settings& settings)
     std::sort(std::begin(files), std::end(files));
 
     test_results total;
-    bool exception_thrown = false;
     for (const auto& f : files)
     {
-        try
-        {
-            const auto res = test_runner{settings}.run_from_file(f);
+        const auto res = test_runner{settings}.run_from_file(f);
 
-            total.passed += res.passed;
-            total.failed += res.failed;
-            total.skipped += res.skipped;
-        }
-        catch (const std::exception& ex)
```

Comment:
`main` has the very same catch all.

Reply:
Without it, it won't outuput `TOTAL` summary string in case of unexpected exception, but perhaps it's not important.

Reply:
I think if there's an exception we shouldn't try to recover, apart from displaying the exception.